.gitignore

```diff
@@ -2,3 +2,8 @@
 node_modules
 dashboard/node_modules
 api/test/fakestats.js
+
+# database and docker files
+docker/Dockerfile-mongodb
```

Comment:
@usmanmuhd 

what are these added in .gitignore?

Reply:
@tarunsinghal92 The first 2 are the files generated by `generate-docker-compose.sh` and the last one is the mongo container data. They both are not needed to be uploaded by anyone. Thought it's better to add it to .gitignore.

Reply:
okay!